default.nix

```diff
@@ -30,22 +30,26 @@ let
         # Various versions of the Nix manuals, grep for (nix-manual)=
         # FIXME: This requires human interaction to update!
         # See ./CONTRIBUTING.md for details.
+        # Using Nix from Nixpkgs could simplify handling, but would requires a bit more setup here
         releases = [
           "2.19"
           "2.18"
           "2.13"
         ];
         inputName = version: pkgs.lib.strings.replaceStrings [ "." ] [ "-" ] version;
-        src = version: (import inputs."nix_${inputName version}").default.doc;
+        src = version: inputs."nix_${inputName version}";
+        manual = version: (import (src version)).default.doc;
         copy = version: ''
-          cp -R ${src version}/share/doc/nix/manual/* $out/manual/nix/${version}
+          cp -R ${manual version}/share/doc/nix/manual/* $out/manual/nix/${version}
+          # use upstream page redirects
+          sed 's#^\(.*\) \(.*\) #/nix/manual/${version}\1 /nix/manual/${version}\2 #g' ${src version}/doc/manual/_redirects >> $out/_redirects
         '';
       in
       with pkgs.lib.strings;
       ''
-        mkdir -p $out/manual/nix/{${concatStringsSep "," releases}}
-        ${concatStringsSep "\n" (map copy releases)}
+        mkdir -p $out/manual/nix/{${concatStringsSep "," releases},}
```

Comment:
Looks like you accidentally added a `,` at the end there. Or is this intentional?

Reply:
Intentional, added comment.